resources/assets/components/Post/index.js

```diff
@@ -55,14 +55,13 @@ class Post extends React.Component {
 
   render() {
     const post = this.props.post;
-    const caption = displayCaption(post);
     const user = this.props.user ? this.props.user : null;
     const signup = this.props.signup;
     const campaign = this.props.campaign;
     const quantity = post.quantity != null ? post.quantity : 0;
     const containerSize = post.type === 'photo' ? '-third' : '-half';
     const textOrCaption = post.type === 'photo' ? 'Photo Caption' : 'Text';
-
+    console.log(post);
```

Comment:
Do we still want this?

Reply:
ah I thought I got rid of that in a prior commit - thanks for flagging!